src/changed.js

```diff
@@ -2,11 +2,9 @@
 
 const buildDepGraph = require('./build-dep-graph');
 const buildChangeGraph = require('./build-change-graph');
-const path = require('path');
 const {
   getWorkspaceCwd,
 } = require('./git');
-const { arePathsTheSame } = require('./changed-files');
```

Comment:
We should remove this export from changed-files.js if it is no longer used.

Reply:
you mean this

Object.assign(module.exports, {
  arePathsTheSame,
});

Reply:
yep. thanks.